src/site/content/en/learn/forms/example/index.md

```diff
@@ -0,0 +1,382 @@
+---
```

Comment:
I'm not going to review this course. I presume it's a copy of something else.

Reply:
yeah i stole this from learn css just to get them started